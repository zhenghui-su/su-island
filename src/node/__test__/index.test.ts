import { expect, test } from 'vitest';

test('add', () => {
  expect(1 + 1).toBe(2);
  expect('map'.slice(1)).toMatchSnapshot();
  expect('map'.slice(1)).toMatchInlineSnapshot('"ap"');
});
