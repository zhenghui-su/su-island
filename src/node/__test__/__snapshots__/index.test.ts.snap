// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`add 1`] = `"ap"`;
